ui/DOMSelection.js

```diff
@@ -78,6 +89,19 @@ DOMSelection.Prototype = function() {
     if (sel.isCollapsed()) {
       wRange.setStart(start.container, start.offset);
       wSel.addRange(wRange);
+      var el = document.createElement("span");
+      el.id = 'cursor';
```

Comment:
'cursor' is common enough to potentially cause conflicts. Something more specific like 'curr-pos-span' is better.